das/simple_das_reader_aggregator.go

```diff
@@ -281,6 +315,10 @@ func (a *SimpleDASReaderAggregator) Start(ctx context.Context) {
 				// Strategy update happens in same goroutine as updates to the stats
 				// to avoid needing extra synchronization.
 				a.strategy.update(a.readers, a.stats)
+			case onlineUrls := <-onlineUrlsChan:
+				if !updateRestfulDasClients(onlineUrls) {
+					return
```

Comment:
Instead of returning, maybe this should just do nothing. Otherwise, a one-time syntax error in the online REST server list would cause this activity to permanently stop.

Reply:
Done.